src/github.js

```diff
@@ -112,6 +112,17 @@ async function assign_reviewers(reviewers) {
   });
 }
 
+// https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
+async function get_team_members(team) {
+  const context = get_context();
+  const octokit = get_octokit();
+
+  return octokit.teams.listMembersInOrg({
+    org: context.repo.org,
+    team_slug: team,
+  })?.map((member) => member.login);
```

Comment:
@hbrysiewicz I haven't confirmed it myself, but I can see multiple examples that tell the response is wrapped in a `data` property 🙂 

https://github.com/Automattic/jetpack/blob/579675983b11c54c092a55bc4e2cf8eb99836df6/projects/github-actions/required-review/src/team-members.js#L39-L45

https://github.com/googleapis/repo-automation-bots/blob/e252b08ac92a33939889a2f1e2093da6727b5140/packages/blunderbuss/src/utils.ts#L296-L301

https://github.com/eslint/cn.eslint.org/blob/bec23372dbd2b36bd2f2ac9169b91df41fd8f35e/_tools/fetch-team-data.js#L66-L70

Search results: https://github.com/search?q=teams.listMembersInOrg&type=code